CONTRIBUTING.md

```diff
@@ -21,3 +21,56 @@ All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
+
+## Getting your pull request through
+### Testing
+An essential part of getting your change through is to make sure all existing tests pass.
+
+#### Prerequisites
+* You will need maven installed to build and run tests.
```

Comment:
s/You will need maven.../[maven](https://maven.apache.org/install.html) to build and run tests.

Reply:
done